contrib/mesos/pkg/executor/executor.go

```diff
@@ -179,6 +166,10 @@ func New(config Config) *KubernetesExecutor {
 	return k
 }
 
+func (k *KubernetesExecutor) InitialRegComplete() <-chan struct{} {
```

Comment:
unfortunately, Registered() is already taken :(

On Tue, Sep 22, 2015 at 4:49 PM, Dr. Stefan Schimanski <
notifications@github.com> wrote:

> In contrib/mesos/pkg/executor/executor.go
> <https://github.com/kubernetes/kubernetes/pull/13036#discussion_r40141777>
> :
>
> > @@ -179,6 +166,10 @@ func New(config Config) *KubernetesExecutor {
> >  	return k
> >  }
> >
> > +func (k *KubernetesExecutor) InitialRegComplete() <-chan struct{} {
>
> or just Registered()
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/13036/files#r40141777>.
>